src/utils/constants.js

```diff
@@ -1,3 +1,4 @@
-const AUTH_STORAGE_KEY = 'wa_cert_authenticated';
-
-export { AUTH_STORAGE_KEY };
+export const AUTH_STORAGE_KEY = 'wa_cert_authenticated';
+export const BASE_URL = 'https://www.googleapis.com/youtube/v3/';
+export const API_TOKEN = 'AIzaSyCU9ohIwX7wOPR2AlUd9nQNMJ716Wb7om0';
+// export const API_TOKEN = 'AIzaSyAJDU0JXSvZylQslHO_cZDwwFmwQh9hWxo';
```

Comment:
It would be better if you keep these variables in a .env file, so this is not exposed in the repository

Reply:
+1